src/ccl_power.c

```diff
@@ -1676,15 +1714,55 @@ double ccl_sigmaR(ccl_cosmology *cosmo,double R, int *status)
     ccl_raise_gsl_warning(gslstatus, "ccl_power.c: ccl_sigmaR():");
     *status |= gslstatus;
   }
-  //TODO: log10 could be taken already in the macros.
-  //TODO: 1E-5 should be a macro
-  //TODO: we should check for integration success
+  // TODO: log10 could be taken already in the macros.
+  // TODO: 1E-5 should be a macro
+  // TODO: we should check for integration success
   gsl_integration_cquad_workspace_free(workspace);
 
-  return sqrt(sigma_R*M_LN10/(2*M_PI*M_PI));
+  return sqrt(sigma_R*M_LN10/(2*M_PI*M_PI))*ccl_growth_factor(cosmo, a, status);
 }
 
+/* --------- ROUTINE: ccl_sigmaV ---------
+INPUT: cosmology, comoving smoothing radius, scale factor
+TASK: compute sigmaV, the variance in the *linear* displacement field 
+smoothed with a tophat filter of comoving size R
+The linear displacement field is the gradient of the linear density field
+*/
+double ccl_sigmaV(ccl_cosmology *cosmo,double R,double a,int *status)
+{
+  SigmaV_pars par;
+  par.status = status;
+
+  par.cosmo=cosmo;
+  par.R=R;
+  gsl_integration_cquad_workspace *workspace=gsl_integration_cquad_workspace_alloc(ccl_gsl->N_ITERATION);
+  gsl_function F;
+  F.function=&sigmaV_integrand;
+  F.params=&par;
+  double sigma_V;
+	int gslstatus = gsl_integration_cquad(&F, log10(ccl_splines->K_MIN_DEFAULT), log10(ccl_splines->K_MAX),
+																				0.0, ccl_gsl->INTEGRATION_SIGMAR_EPSREL,
+																				workspace,&sigma_V,NULL,NULL);
+
+  if(gslstatus != GSL_SUCCESS) {
+    ccl_raise_gsl_warning(gslstatus, "ccl_power.c: ccl_sigmaV():");
+    *status |= gslstatus;
+  }
+
+  // TODO: log10 could be taken already in the macros.
+  // TODO: 1E-5 should be a macro
```

Comment:
Address these TODO items.

Reply:
I am not entirely sure what these TODO items are for in sigmaR (I forget who wrote ccl_power off the top of my head), but I believe both of these look like they are already resolved.

Reply:
Apparently this is sorted, so I've now done